common/djangoapps/student/views.py

```diff
@@ -2552,13 +2556,18 @@ def password_reset_confirm_wrapper(request, uidb36=None, token=None):
 
     if request.method == 'POST':
         password = request.POST['new_password1']
-        is_password_valid, password_err_msg = validate_password(user, password)
-        if not is_password_valid:
+        is_security_violated, password_err_msg = validate_password(user, password)
```

Comment:
Is this the only place we call `valid_password()` function? You made a change to the interface of the function. Just be extra careful with it.